src/console/workerInterface.ts

```diff
@@ -20,6 +20,8 @@ export const mirrorWorker: Promise<Thread & MirrorWorkerMethods> = new Promise((
       });
       worker.getErrorObservable().subscribe((errorMessage) => {
         mirrorLog.error(errorMessage);
+        const message = errorMessage instanceof Error ? errorMessage.message : errorMessage;
+        ipc_consoleMirrorErrorMessageEvent.main?.trigger({ message }).catch(mirrorLog.error);
```

Comment:
```suggestion
      worker.getErrorObservable().subscribe((err) => {
        mirrorLog.error(err);
        const message = err.message ?? JSON.stringify(err);
        void ipc_consoleMirrorErrorMessageEvent.main!.trigger({ message });
```

I actually think using `void` when sending messages is pretty okay since if it errors, it's usually an error outside our control. Likewise asserting that the main attribute exists when calling from the main thread.
And then the `err.message ?? JSON.stringify(err)` pattern is used elsewhere in the codebase so we should stick with that instead of switching on `instanceOf Error`.

Reply:
so the reason I use the instanceof method is because we have a string | Error type, so we have to do a type check.

Reply:
In that case that's fine to use the `instanceof`. But I would still prefer the optional chaining `?` to be replaced with non-null assertion `!`.

Reply:
whoops, i didn't even notice that I used an optional chaining there, fixed.